### PR TITLE
[vividus-tests] Add integration tests for composite step with comment

### PR DESCRIPTION
**Pull request by VitaliLebedzeu**
Relates #649

### CODE REVIEW

File: vividus-tests/src/main/resources/steps/defaults/field.steps [hunk 1]
```diff
@@ -1,3 +1,7 @@
 Composite: Then field value is `$expected`
 When I perform javascript 'return document.querySelector('#textInput').value' and save result to the 'Scenario' variable 'fieldValue'
 Then `${fieldValue}` is equal to `<expected>`
+
+Composite: When composite with comment
+!-- comment
+When I initialize the scenario variable `` with value `value`
```
Comment (valfirst) [inline vividus-tests/src/main/resources/steps/defaults/field.steps:7]:
```
Composite: When I run composite step with comment
When I initialize the scenario variable `before-comment` with value `before`
!-- comment
When I initialize the scenario variable `after-comment` with value `after`
```

File: vividus-tests/src/main/resources/story/integration/JbehaveCore.story [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Description: Integration tests for JBehave core.
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/JbehaveCore.story:1]: rename the story to `CompositeSteps.story`

Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/JbehaveCore.story:1]: `Description: Integration tests for composite steps`

File: vividus-tests/src/main/resources/story/integration/JbehaveCore.story [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/JbehaveCore.story:2]: add meta with `@epic` and `@feature` on story-level:
```
Meta:
    @epic vividus-bdd-engine
    @feature composite-steps
```

File: vividus-tests/src/main/resources/story/integration/JbehaveCore.story [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+Scenario: Verify composite step with comment
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/JbehaveCore.story:3]: add meta with `@issueId` on scenario level

File: vividus-tests/src/main/resources/steps/defaults/field.steps (hunk 1 above)
Comment (VitaliLebedzeu) [reply vividus-tests/src/main/resources/steps/defaults/field.steps:7]: fixed

File: vividus-tests/src/main/resources/story/integration/JbehaveCore.story (hunk 2 above)
Comment (VitaliLebedzeu) [reply vividus-tests/src/main/resources/story/integration/JbehaveCore.story:1]: fixed

Comment (VitaliLebedzeu) [reply vividus-tests/src/main/resources/story/integration/JbehaveCore.story:1]: fixed

File: vividus-tests/src/main/resources/story/integration/JbehaveCore.story (hunk 3 above)
Comment (VitaliLebedzeu) [reply vividus-tests/src/main/resources/story/integration/JbehaveCore.story:2]: fixed

File: vividus-tests/src/main/resources/story/integration/JbehaveCore.story (hunk 4 above)
Comment (VitaliLebedzeu) [reply vividus-tests/src/main/resources/story/integration/JbehaveCore.story:3]: fixed

File: vividus-tests/src/main/resources/story/integration/CompositeSteps.story [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+Meta:
+    @epic vividus-bdd-engine
+    @feature composite-steps
+
+Scenario: Verify composite step with comment
+Meta:
+    @issueId 649
+When composite with comment
+Then `${}` is equal to `value`
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/CompositeSteps.story:11]: you need to update the step and validations according to the changes in the composite step implementation

Comment (VitaliLebedzeu) [reply vividus-tests/src/main/resources/story/integration/CompositeSteps.story:11]: Yep. I've just run the tests

**Comment by VitaliLebedzeu** [comment]
fixed... pushed

**Pull request merged**